src/api/app/assets/javascripts/webui/responsive_ux.js

```diff
@@ -3,14 +3,27 @@ $(function () {
     $('a[data-toggle="offcanvas"]').toggleClass('active');
     $('.offcanvas-collapse').toggleClass('open');
     $('.watchlist-collapse').removeClass('open');
+    $('.contextual-actions-collapse').removeClass('open');
     $('a[data-toggle="watchlist"]').removeClass('active');
+    $('a[data-toggle="contextual-actions"]').removeClass('active');
   });
 
   $('[data-toggle="watchlist"]').on('click', function () {
     $('a[data-toggle="watchlist"]').toggleClass('active');
     $('.watchlist-collapse').toggleClass('open');
     $('.offcanvas-collapse').removeClass('open');
+    $('.contextual-actions-collapse').removeClass('open');
     $('a[data-toggle="offcanvas"]').removeClass('active');
+    $('a[data-toggle="contextual-actions"]').removeClass('active');
+  });
+
+  $('[data-toggle="contextual-actions"]').on('click', function () {
```

Comment:
I'm on the fence if I should DRY this or not. What do you think?

Reply:
Go for it!